File: ProyectoBaseWebAppSaul/app/CodigoAserrador.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class CodigoAserrador extends Model
{
    protected $table = 'codigos_aserradores';
    protected $fillable = [
        'aserrador_id', 'codigo', 'empresa_id', 'descripcion', 'estado', 'creador_id',
        'modificador_id'
    ];
    public function aserrador()
    {
        return $this->belongsTo('App\Aserrador');
    }
    public function scopeActive($query)
    {
        return $query->where('estado', 'A');
    }
}

File: CosechaApp-main/app/src/main/java/com/techtraining/cosechasapp/db/CodigoAserrador.java
package com.techtraining.cosechasapp.db;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;
@Entity
public class CodigoAserrador {
    @PrimaryKey
    public int id;

    @ColumnInfo(name = "descripcion")
    public String descripcion;

    @ColumnInfo(name = "codigo")
    public String codigo;

    @ColumnInfo(name = "aserrador_id")
    public int aserradorId;

    @ColumnInfo(name = "empresa_id")
    public int empresaId;

    @ColumnInfo(name = "estado")
    public String estado;
}
